### PR TITLE
[plugin.video.catchuptvandmore] 0.2.31

### PR DESCRIPTION
**Pull request by sy6sy2**
### Description

[Version 0.2.31]

[CA - TVA] Fix Catch-up TV
[CH - SRGSSR] Fix Catch-up TV
[ALL] Add a notification "Plugin CUTVMore non compatible Kodi 19"

[Version 0.2.30]

[BE - ATV] Add Live TV
[BE - BRUZZ] Add Live TV
[BE - Focus and WTV] Add Live TV
[BE - Rob TV] Add Live TV
[BE - TVL] Add Live TV
[BE - TVOost] Add Live TV
[BE - RTLPlay] Move to Android API
[BE - RTBF] Fix Multi Lives
[TN - watania1/watania2] Fix Live TV
[BE - Telesambre] Fix Live TV
[BE - ACTV] Fix Live TV
[FR - TF1 Thematique] Fix Catch-up TV (Histoire, Breizh, Ushuaia)
[FR - Culturebox] Add Live TV
[FR - CNews] Possible fix to update video list
[Live TV] Add IPTV Manager support
[BE - HNL] Add Live TV
[FR - Equipe TV] Fix Catch-up TV
[BE - Fosdem] Add contents for 2021
[FR - France TV] Fix Live TVs
[BE - TVCom] Fix Live TV
[FR - MyCanal] Fix missing image in MyCanal
[UK - STV] Fix Catch-up TV
[UK - Blaze TV] Fix Catch-up TV
[CA - TeleQuebec] Readd Live TV
[CA - TVA] Fix for Catch-up TV of Star Academie

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.catchuptvandmore/resources/lib/channels/be/tvl.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# It makes string literals as unicode like in Python 3
+from __future__ import unicode_literals
+
+from codequick import Resolver
+
+
+@Resolver.register
+def get_live_url(plugin, item_id, **kwargs):
+
+    return 'http://api.new.livestream.com/accounts/27755193/events/8452383/live.m3u8'
```
Comment (enen92) [inline plugin.video.catchuptvandmore/resources/lib/channels/be/tvl.py:34]: where does ths one come from?

Comment (enen92) [reply plugin.video.catchuptvandmore/resources/lib/channels/be/tvl.py:34]: nvm, confirmed legit

File: plugin.video.catchuptvandmore/resources/lib/menu_utils.py [hunk 2]
```diff
@@ -162,6 +162,9 @@ def get_sorted_menu(plugin, menu_id):
     current_menu = importlib.import_module('resources.lib.skeletons.' +
                                            menu_id).menu
 
+    if menu_id == "root":
+        xbmcgui.Dialog().ok("Information : Kodi 19 - Matrix", "Le plugin Catch-Up TV and More n'est pas compatible avec Kodi 19. Nous sommes en attente de mise à jour de plugin tièrs utilisés dans ce plugin. Si vous voulez continuer à utiliser ce plugin, la dernière version supportée est le Kodi 18.9.", "The Catch-Up TV and More plugin is not compatible with Kodi 19. We are awaiting updates to third party plugins used in this plugin. If you want to continue using this plugin, the last supported version is Kodi 18.9.")
```
Comment (enen92) [inline plugin.video.catchuptvandmore/resources/lib/menu_utils.py]: localized pls

**Review by enen92 — CHANGES_REQUESTED** [review]
Minor and good to go

**Comment by sy6sy2** [comment]
Done 😉

**Pull request merged**